client/lib/analytics/hotjar.js

```diff
@@ -7,7 +7,8 @@ import debug from 'debug';
  * Internal dependencies
  */
 import config from 'config';
-import { mayWeTrackCurrentUserGdpr, doNotTrack, isPiiUrl } from './utils';
+import { mayWeTrackCurrentUserGdpr, isPiiUrl } from './utils';
+import { doNotTrack } from '@automattic/calypso-analytics';
```

Comment:
I think this line should move to `External dependencies`. Also appears in a few other places.

Reply:
I'm just checking that p4TIVU-9lc-p2